libs/journeys/ui/src/components/StepHeader/StepHeader.tsx

```diff
@@ -84,7 +86,9 @@ export function StepHeader({
                 {journey?.displayTitle ?? journey?.seoTitle}
               </Typography>
             </Stack>
+
             {/* Menu */}
+            <StepHeaderMenu />
```

Comment:
Remove comment and space
```suggestion
            <StepHeaderMenu />
```